### PR TITLE
Update plugin for Blender 2.90 and add support for modifiers

### PR DESCRIPTION
**Pull request by tpdickso**
This PR adds a menu item for `V-HACD` to the `Object` menu in the
3D view of blender, as starting with version 2.90, Blender won't show
operators in the operator search unless they've got a corresponding
menu entry.

This also adds a checkbox to the operator settings that allows the user
to generate geometry based on the mesh after applying all moderators,
rather than its shape in edit mode. This is useful when mesh geometry is
changed significantly by deformation modifiers.

### CODE REVIEW

File: object_vhacd.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ class VHACD_OT_VHACD(bpy.types.Operator):
     bl_idname = 'object.vhacd'
     bl_label = 'Convex Hull (V-HACD)'
     bl_description = 'Create accurate convex hulls using Hierarchical Approximate Convex Decomposition'
-    bl_options = {'REGISTER', 'PRESET'} # PRESET doesn't seem to require AdPresetBase operator to work anymore...
+    bl_options = {'REGISTER', 'PRESET', 'UNDO'} # PRESET doesn't seem to require AdPresetBase operator to work anymore...
```
Comment (tpdickso) [inline object_vhacd.py:152]: Let me know if this is unwanted; I've found it useful to be able to undo generating the meshes.

**Comment by tpdickso** [comment]
Screenshot from 2020-09-13 13-29-39
Screenshot from 2020-09-13 13-29-49
Here's an example of the difference in generated geometry between the base geometry for this rock object, and the result of Subdivision, Displace, Decimate and Bevel modifiers to give it a rocky surface.

Credit to [CG Geek](https://www.youtube.com/watch?v=4EqLyGsu3AA) for the technique.

**Comment by stuarta0** [comment]
Works well, would be great to see this merged

**Comment by tpdickso** [comment]
Hey! Just wanted to bump this. I would not mind helping to maintain this repository, if it's beneficial.

**Pull request merged**